[2024-12-15T23:46:26.089+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: covid_data_pipeline.validate_data __airflow_temporary_run_2024-12-15T23:39:03.658444+00:00__ [queued]>
[2024-12-15T23:46:26.110+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: covid_data_pipeline.validate_data __airflow_temporary_run_2024-12-15T23:39:03.658444+00:00__ [queued]>
[2024-12-15T23:46:26.113+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2024-12-15T23:46:26.140+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): validate_data> on 2024-01-01 00:00:00+00:00
[2024-12-15T23:46:26.150+0000] {standard_task_runner.py:57} INFO - Started process 307 to run task
[2024-12-15T23:46:26.155+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'covid_data_pipeline', 'validate_data', '__***_temporary_run_2024-12-15T23:39:03.658444+00:00__', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/covid_pipeline_dag.py', '--cfg-path', '/tmp/tmpnfsmgv_z']
[2024-12-15T23:46:26.160+0000] {standard_task_runner.py:85} INFO - Job 11: Subtask validate_data
[2024-12-15T23:46:26.248+0000] {task_command.py:416} INFO - Running <TaskInstance: covid_data_pipeline.validate_data __airflow_temporary_run_2024-12-15T23:39:03.658444+00:00__ [running]> on host 6cd16bdff3c4
[2024-12-15T23:46:26.375+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='ctw2012105@@gmail.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='covid_data_pipeline' AIRFLOW_CTX_TASK_ID='validate_data' AIRFLOW_CTX_EXECUTION_DATE='2024-01-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='__***_temporary_run_2024-12-15T23:39:03.658444+00:00__'
[2024-12-15T23:46:26.413+0000] {data_validator.py:34} INFO - Validating data schema
[2024-12-15T23:46:26.414+0000] {data_validator.py:50} INFO - Validating data quality
[2024-12-15T23:46:26.419+0000] {data_validator.py:82} INFO - All validations passed successfully
[2024-12-15T23:46:26.420+0000] {python.py:194} INFO - Done. Returned value was: Data validation completed
[2024-12-15T23:46:26.466+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=covid_data_pipeline, task_id=validate_data, execution_date=20240101T000000, start_date=20241215T234626, end_date=20241215T234626
[2024-12-15T23:46:26.539+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-12-15T23:46:26.575+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
